### ISSUE TITLE
VS Code Version?

### ISSUE THREAD

**Issue by kurtextrem** [simurai/one-dark-vivid-syntax#7]
Is there anything regarding VS Code planned? :)

**Comment by simurai** [comment]
Nothing planned, but there is https://marketplace.visualstudio.com/items?itemName=akamud.vscode-theme-onedark that might be close?

**Comment by kurtextrem** [comment]
Those aren't the vivid colors, aren't they?

**Comment by simurai** [comment]
Yeah, probably from the original. But at least the logic should be somewhat close.

**Comment by hadicodes** [comment]
Hi Simurai!

Love love love your One Dark Vivid theme for atom; Thank you for that.

Could you please make the same theme for VSCode?

Hadi
